custom_components/adaptive_lighting/switch.py

```diff
@@ -1358,7 +1314,7 @@ async def _update_manual_control_and_maybe_adapt(
                 else:
                     _fire_manual_control_event(self, light, context)
             else:
-                await self._adapt_light(light, transition, force=force, context=context)
+                await self._adapt_light(light, transition, context=context)
```

Comment:
Why is the `force` argument no longer passed through here?

Reply:
`force` controlled whether `_adapt_light` should send adaptation commands when the known entity state equals the requested state, and that is now decided by `experimental_skip_redundant_commands`.

I couldn't find out why `force` was enforced in particular situations and why it would be necessary, e.g. https://github.com/basnijholt/adaptive-lighting/blob/36c656b7958f6a9a637b50800534749038e146ee/custom_components/adaptive_lighting/switch.py#L527-L537
and I was hoping you'd know :)

Reply:
We pass `force=True` when toggling sleep mode for example.

Reply:
Between all the changes made by @th3w1zard1 and my old code, I can also no longer figure out why that was needed 😅